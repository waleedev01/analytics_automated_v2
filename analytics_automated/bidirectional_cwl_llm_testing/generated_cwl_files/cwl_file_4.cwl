cwlVersion: v1.0
class: CommandLineTool

baseCommand: echo

outputs:
  output_file:
    type: stdout

requirements:
  InlineJavascriptRequirement: {}

inputs:
  input_file:
    type: File

outputs:
  output_file:
    type: File
    outputBinding:
      glob: "output.txt"