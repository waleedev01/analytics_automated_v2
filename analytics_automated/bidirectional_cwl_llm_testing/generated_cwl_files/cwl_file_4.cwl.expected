{"is_valid": false, "error": "Validation failed: Missing 'inputs' in CWL file"}